plugins/main/macvlan/macvlan_test.go

```diff
@@ -225,544 +243,560 @@ var _ = Describe("macvlan Operations", func() {
 		Expect(targetNS.Close()).To(Succeed())
 		Expect(testutils.UnmountNS(targetNS)).To(Succeed())
 	})
+	for _, inContainer := range []bool{false, true} {
+		isInContainer := inContainer
+		masterInterface := MASTER_NAME
+		if inContainer {
+			masterInterface = MASTER_NAME_INCONTAINER
+		}
+		for _, ver := range testutils.AllSpecVersions {
+			// Redefine ver inside for scope so real value is picked up by each dynamically defined It()
+			// See Gingkgo's "Patterns for dynamically generating tests" documentation.
+			ver := ver
+
+			It(fmt.Sprintf("[%s] creates an macvlan link in a non-default namespace", ver), func() {
+				conf := &NetConf{
+					NetConf: types.NetConf{
+						CNIVersion: ver,
+						Name:       "testConfig",
+						Type:       "macvlan",
+					},
+					Master:     masterInterface,
+					Mode:       "bridge",
+					MTU:        1500,
+					LinkContNs: isInContainer,
```

Comment:
Could you please add test case without 'LinkContNs' and verify default behavior (i.e. without 'LinkContNs') is the same as 'LinkContNs' is `false`?

Reply:
Done